linux/datadog_wrapper

```diff
@@ -132,6 +135,25 @@ setUpDotnetEnv() {
     export DD_DOTNET_TRACER_HOME="${DD_DIR}"
 }
 
+setupPHPEnv() {
+    echo "Setting up Datadog tracing for PHP"
+    getBinaries || return
+
+    if [ -z "$DD_PHP_TRACER_VERSION" ]; then
+        DD_PHP_TRACER_VERSION=0.83.1
+    fi
+
+    DD_PHP_TRACER_URL=https://github.com/DataDog/dd-trace-php/releases/download/${DD_PHP_TRACER_VERSION}/datadog-setup.php
+
+    echo "Installing PHP tracer from $DD_PHP_TRACER_URL"
+    if curl -LO --fail "$DD_PHP_TRACER_URL"; then
+        eval "php datadog-setup.php --php-bin=all"
```

Comment:
Is the `eval` really necessary? Honest question.

Reply:
Yes. I was running into issues where the string was not being executed as a command correctly. Using eval allowed the command to execute as intended.